configs/env_var.sh

```diff
@@ -0,0 +1,6 @@
+export TWITTER_CONSUMER_KEY=""
```

Comment:
Duplicate file. Should be removed anyways.

Reply:
it's a skeleton script that we can copy and edit when deploying (for reducing migration cost :) )